src/components/shell.css

```diff
@@ -58,66 +60,75 @@
   height: var(--shell-header-height);
   box-sizing: border-box;
   position: fixed;
-  /* left: calc(50vw - var(--shell-control-icon-width) - var(--shell-control-icon-width) / 2); */
-  left: 10vw;
-  width: 80vw;
   overflow: hidden;
   display: flex;
   justify-content: space-between;
   align-items: center;
   padding: var(--xsgrd) var(--sgrd);
 
-  @media (--narrow-screen) {
-    left: 0;
-    width: 100vw;
-  }
   & .shell__controls__left {
     display: flex;
     align-items: center;
     flex: 1 1 50%;
   }
-  & .shell__controls__right {
```

Comment:
This diff makes this more confusing than it is: all the `.shell__controls*` are now wrapped in `@media (--narrow-screen)`